### PR TITLE
[iptables] Regenerate test expectations

### PR DESCRIPTION
**Pull request by aleksmaus**
## Proposed commit message

Regenerate test expectations that are failing now. 
Not even sure if we suppose to release the version of the package for this kind of thing. Will place the PR and will see what happens.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes https://github.com/elastic/integrations/issues/10757

### CODE REVIEW

**Comment by aleksmaus** [comment]
Tested on few latest versions of the stack, that's what we get for this IP resolution now


Image


While the test expectation was 
```
-                "geo": {
-                    "continent_name": "Asia",
-                    "country_iso_code": "BT",
-                    "country_name": "Bhutan",
-                    "location": {
-                        "lat": 27.5,
-                        "lon": 90.5
```

**Comment by aleksmaus** [comment]
Hmm, it's odd, it fails in CI now with these regenerated files, while it passes against the snapshots of 8.14, 8.15, 8.16 that you get with the cloud deployment. 

```
test case failed: Expected results are different from actual ones: --- want
+++ got
@@ -80,9 +80,15 @@
             },
             "source": {
                 "as": {
-                    "number": 35908,
-                    "organization": {
-                        "name": "VPLSNET"
+                    "number": 35908
+                },
+                "geo": {
+                    "continent_name": "Asia",
+                    "country_iso_code": "BT",
+                    "country_name": "Bhutan",
+                    "location": {
+                        "lat": 27.5,
+                        "lon": 90.5
```

@andrewkroh @taylor-swanson How do you guys go about reproducing the exact environment for geoip processors, so you can get the proper test expectations generated that line up?

**Comment by andrewkroh** [comment]
elastic-package is supposed to setup ES with a custom DB to ensure reproducibility. Did something change where this DB is not be loaded properly?

https://github.com/elastic/elastic-package/blob/main/docs/howto/ingest_geoip.md 


-----

If the change is only affecting test files (no user facing changes) then we shouldn't need to make a release, so could omit the manifest/changelog edits.

File: packages/iptables/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.16.2"
+  changes:
+    - description: Regenarate geoip related tests expectations
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/8888888
```
Comment (andrewkroh) [inline packages/iptables/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/10871
```

**Comment by aleksmaus** [comment]
> elastic-package is supposed to setup ES with a custom DB to ensure reproducibility. Did something change where this DB is not be loaded properly?

Ahh, that's good to know. That explains why the different results, then. Don't see the same errors with the local tests runs. Will close this PR then and dig more for repro locally.

**Pull request closed without merging**